### PR TITLE
az-storage: add page

### PR DESCRIPTION
**Pull request by WhileLoop**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by blueskyson — APPROVED** [review]
Looks good to me. Only a little suggestion, move `az storage account create` to the first example because it seems to be a beginning step of this application.

File: pages/common/az-storage.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+- List the access keys for a storage account:
+
+`az storage account keys list -g {{group_name}} -n {{account_name}}`
+
+- Delete a storage account:
+
+`az storage account delete -g {{group_name}} -n {{account_name}}`
+
+- Update storage account min tls setting:
```
Comment (sbrl) [inline pages/common/az-storage.md:23]: Perhaps this will read better?
```suggestion
- Update the minimum tls version setting for a storage account:
```

Comment (WhileLoop) [reply pages/common/az-storage.md]: This has been updated.

**Review by sbrl — APPROVED** [review]
Thanks, @WhileLoop!

**Pull request merged**